app/assets/stylesheets/css/fields/tags.css

```diff
@@ -10,7 +10,7 @@ tags.tagify {
 }
 
 tags.tagify {
-  @apply !p-1 !gap-1 !border-gray-200;
```

Comment:
The only thing that I would add here is `!px-2`. This makes the horizontal padding the same as the vertical one.

### Before

![CleanShot 2024-03-19 at 07 32 12@2x](https://github.com/avo-hq/avo/assets/1334409/355e9011-ade1-4978-acf7-21ec6359910b)

### After
![CleanShot 2024-03-19 at 07 31 58@2x](https://github.com/avo-hq/avo/assets/1334409/5564aef0-c12c-45b9-9b6a-bfa18811a14b)